src/config.ts

```diff
@@ -1,32 +1,44 @@
 import HeadMRAThumbnail from '@/src/assets/samples/head-mra.jpg';
 import LiverCTThumbnail from '@/src/assets/samples/liver-ct.jpg';
 import AbdomenMRIThumbnail from '@/src/assets/samples/abdomen-mri.jpg';
-import { Layout, LayoutDirection, ViewConfig, ViewKey } from './store/views';
+import { Layout, LayoutDirection, LayoutView } from './types/layout';
 
-export const Views: Record<string, ViewConfig> = {
+export const Views: Record<string, LayoutView> = {
   Coronal: {
-    objType: 'View2D',
-    key: ViewKey.CoronalView,
-    viewDirection: 'Right',
-    viewUp: 'Superior',
+    objType: 'View',
+    viewType: 'View2D',
+    id: 'Coronal',
+    props: {
+      viewDirection: 'Right',
+      viewUp: 'Superior',
+    },
   },
   Sagittal: {
-    objType: 'View2D',
-    key: ViewKey.SagittalView,
-    viewDirection: 'Posterior',
-    viewUp: 'Superior',
+    objType: 'View',
+    viewType: 'View2D',
+    id: 'Sagittal',
+    props: {
+      viewDirection: 'Posterior',
+      viewUp: 'Superior',
```

Comment:
So I think these values should be include in our state file. Should we add them to the view store, rather than just the id? Obvious we can just look them up based on the key, but from a reproducible standpoint I think they belong in the state file.

Reply:
The current layout should be available from the view store. I think a non-cloned reference to one of the layouts gets saved into the view store.